docs/developer-guide/building-flashing.md

```diff
@@ -38,7 +38,7 @@ To build only the F4 firmware, use `make BOARD=REVO`. To build only the F1 firmw
 
 ## Flashing Newly-Built Firmware
 
-First, make sure you have configured your computer as described in the [Serial Port Configuration](/user-guide/flight-controller-setup/#serial-port-configuration) section of the user guide.
+First, make sure you have configured your computer as described in the [Serial Port Configuration](../user-guide/flight-controller-setup.md#serial-port-configuration) section of the user guide.
```

Comment:
curious as to why this change was made, since the browser interprets it already as
http://127.0.0.1:8000/user-guide/flight-controller-setup/#serial-port-configuration
which is what it used to be. Is this a 'best practices', preference, or something else?
Similar changes were made in User-Guide: Getting Started. I don't mean to come across as rude, just want to learn.

Reply:
Yeah it's just a best practices thing; that's how the mkdocs documentation says to do internal links